docs/usage/logging.md

```diff
@@ -0,0 +1,126 @@
+# Logging stack
+
+### When to use the logging stack?
+Currently Kubernetes uses the underlying container runtime logging which does not persist logs for not existing containers. This makes it difficult to investigate issues occurred in long ago destroyed containers.
+
+
+### Components:
+* A Fluent-bit daemonset which works like a log collector and custom custom Golang plugin which spreads log messages to their Loki instances
+* One Loki Statefulset in the `garden` namespace which contains logs for the seed cluster and one per shoot namespace which contains logs for shoot's controlplane.
+* One Grafana Deployment in `garden` namespace and two Deployments per shoot namespace (one exposed to the end users and one for the operators). Grafana is the UI component used in the logging stack.
+
+### How to access the logs
+The first step is to authenticate in front of the Grafana ingress. The secret with the credentials can be found in garden-<project> namespace under <shoot-name>.monitoring.
+After that there are two options for monitoring logs. The URLs can be found in the Gardener Dashboard in `Monitoring and Logging` section.
+* The first one is to log in in the admin panel (usually located at the bottom left corner). The default username and password are `admin`, `admin` (After successful log in, you will be asked for changing the default password. It should not be changed because this dashboard will be used from all of the operators!). After that a new `Explore` menu will be available (left side of the screen). It is used for creating a log queries using the predefined filters in Loki. For example: 
```

Comment:
```suggestion
* The first one is to log in in the admin panel located in bottom left corner. The default username and password are `admin`, `admin`. After successful log in, you will be asked to change the default password. **These credentials are shared by all operators. If you change the password this will affect their access.** After that, a new `Explore` menu will be available at the left side of the screen. It is used for creating log queries using the predefined filters in Loki. For example: 
```